lib/schema_associations/active_record/associations.rb

```diff
@@ -205,6 +225,10 @@ def _determine_association_names(reference_name, referencing_name, references_na
           { :belongs_to => belongs_to.to_sym, :has_one => has_one.to_sym, :has_many => has_many.to_sym }
         end
 
+        def _deferred_associations
```

Comment:
For memory considerations, I'd recommend unsetting the variable when the deferred associations are done so it can be GC'ed. a large app can have hundreds/thousands of relations.

Reply:
Also, adding another method 🤔 if we can avoid it that would be great so we won't end up with name collisions.